### DISCUSSION TITLE
How to set meta values (author, url, description, name) on cli training?

### DISCUSSION THREAD

**Discussion by info2000** [explosion/spaCy#11454] [Help: Coding & Implementations]
I had models trained from the cli process that next are implemented loading the model folder into the code.
The issue:
How can I set the meta values into the config file that on my case is unique per model, to these values are inserted on the meta.json auto-generated?
 

Thanks

**Comment by polm** [comment]
You can't pass those fields directly to CLI training. What you can do is read in the generated `meta.json`, modify any fields you're interested in, and write it back out. 

If you have a `meta.json` already, you can also specify it when using `spacy package`, see [here](https://spacy.io/usage/saving-loading/#models-generating).
